src/components/security/SecurityPage.js

```diff
@@ -0,0 +1,172 @@
+      {
+        issueType: 'type1',
+        description: 'test1',
+        subject: {tenantId: 'tenant1'},
+        regionId: 'region1'
+      },
+      {
+        issueType: 'type2',
+        description: 'test2',
+        subject: {tenantId: 'tenant2'},
+        regionId: 'region2'
+      },
+      {
+        issueType: 'securityGroup',
+        description: 'Security group too open',
+        subject: {tenantId: 'demo'},
+        regionId: 'region1'
+      }
+    ];
+    transaction(() => {
+      securityData.update(dataPeriod, responseBody);
+    });
+  }
+
+  fetchData() {
+    return this.constructor.fetchData(this.props, this.props);
+  }
+
+  async changeDataPeriod(dataPeriod) {
+    await this.constructor.fetchData(this.props, this.props, {dataPeriod});
+    this.props.uiState.activeStatusDataPeriod = dataPeriod;
+  }
+
+  filters = [
+    {
+      name: 'type',
+      title: 'Any type',
+      match: (issue, value) => issue.issueType === value
+    },
+    {
+      name: 'tenant',
+      title: 'Any tenant',
+      match: (issue, value) => issue.subject.tenantId === value
+    },
+    {
+      name: 'search',
+      title: 'Full text search',
+      match: (issue, value) => issue.issueType.indexOf(value) >= 0 ||
+        issue.subject.tenantId.indexOf(value) >= 0 ||
+        issue.description.indexOf(value) >= 0
+    }
+  ];
+
+  @observable filterValues = {
+    type: '',
+    tenant: '',
+    search: ''
+  };
+
+  @computed get filterOptions() {
+    return {
+      type: this.props.securityData.getIssueTypes(this.props.uiState.activeStatusDataPeriod),
+      tenant: this.props.securityData.getTenants(this.props.uiState.activeStatusDataPeriod)
+    };
+  }
+
+  changeFilter = (name, value) => {
+    this.filterValues[name] = value;
+  }
+
+  render() {
+    const {params: {regionName}, uiState: {activeStatusDataPeriod}, securityData} = this.props;
+    const issues = securityData.get(activeStatusDataPeriod).issues;
+    const filteredIssues = issues.filter(
+      (issue) => every(this.filters,
+        ({name, match}) => !this.filterValues[name] || match(issue, this.filterValues[name])
+      )
+    );
+
+    return (
+      <div>
+        <div className='container-fluid'>
+          <h1>{'Security: ' + (regionName || 'All Regions')}</h1>
+          <div className='btn-toolbar'>
+            <div className='filters'>
+              {this.filters.map((filter) =>
+                <DataFilter
+                  key={filter.name}
+                  {...filter}
+                  value={this.filterValues[filter.name]}
+                  options={this.filterOptions[filter.name]}
+                  onChange={this.changeFilter}
+                  disabled={!issues.length}
+                />
+              )}
+            </div>
+            <StatusDataPeriodPicker
+              className='pull-right'
+              onDataPeriodChange={(dataPeriod) => this.changeDataPeriod(dataPeriod)}
+              disabled={!issues.length}
```

Comment:
Why? They can exist here for another period - for example, there are no issues for a day, but there are some issues for month